source/manual/deploying-terraform.html.md

```diff
@@ -9,77 +9,80 @@ review_in: 3 months
 ---
 
 We use [Terraform](https://terraform.io) for configuring GOV.UK infrastructure in AWS.
-This document describes:
-
- * How to set up your machine to run Terraform.
- * How to deploy infrastructure changes using Terraform.
 
 ## One-time setup
 
 ### 1. Check that you have sufficient access
 
 Which changes you can deploy depends on the level of access you have
-to our AWS environments. Specifically, the level of access your Amazon Resource Name (ARN) has been given.
+to our AWS environments.
 
-- `govuk-users` (`role_user_user_arns`) can't deploy anything.
-- `govuk-powerusers` (`role_poweruser_user_arns`) and `govuk-platformhealth-powerusers` (`role_platformhealth_poweruser_user_arns`) can deploy everything except IAM (users and policies).
-- `govuk-administrators` (`role_admin_user_arns`) and `govuk-internal-administrators` (`role_internal_admin_user_arns`) can deploy everything including IAM.
+- `govuk-users` can't deploy anything.
+- `*-powerusers` can deploy anything except IAM.
+- `*-administrators` can deploy anything.
 
-You can find which class of user you are (that is, which roles your ARN has been assigned to) in the
-[infra-security project in govuk-aws-data](https://github.com/alphagov/govuk-aws-data/tree/master/data/infra-security).
+You can find which class of user you are [infra-security project in govuk-aws-data](https://github.com/alphagov/govuk-aws-data/tree/master/data/infra-security).
 
-### 2. Install `gds-cli`
+## Deploying on Jenkins
 
-See [instructions for installing `gds-cli`](/manual/access-aws-console.html).
+> **This is not the recommended approach.**
+>
+> It's slightly easier to start with, but requires you to push your changes to GitHub and you'll quickly want to switch to deploying locally (see the next section) to avoid pasting secrets over and again!
```

Comment:
```suggestion
> It's slightly easier to start with, but you'll quickly want to switch to deploying locally (see the next section) to avoid pasting secrets over and again!
```

Every one of these approaches requires pushing to GitHub.

Reply:
Oops - fixed 👍